bundles/org.openhab.binding.vdr/src/main/java/org/openhab/binding/vdr/internal/VDRHandler.java

```diff
@@ -0,0 +1,347 @@
+                                result = OnOffType.ON;
+                            } else {
+                                result = OnOffType.OFF;
+                            }
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_VOLUME:
+                            SVDRPVolume volume = con.getSVDRPVolume();
+                            result = new PercentType(volume.getVolume());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CHANNEL:
+                            SVDRPChannel channel = con.getCurrentSVDRPChannel();
+                            result = new DecimalType(channel.getNumber());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CHANNEL_NAME:
+                            SVDRPChannel svdrpChannel = con.getCurrentSVDRPChannel();
+                            result = new StringType(svdrpChannel.getName());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_POWER:
+                            SVDRPDiskStatus status = con.getDiskStatus();
+                            if (status.getPercentUsed() >= 0) {
+                                result = OnOffType.ON;
+                            } else {
+                                result = OnOffType.OFF;
+                            }
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_DISKUSAGE:
+                            SVDRPDiskStatus diskStatus = con.getDiskStatus();
+                            result = new DecimalType(diskStatus.getPercentUsed());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CURRENT_EVENT_TITLE:
+                            entry = con.getEpgEvent(SVDRPEpgEvent.TYPE.NOW);
+                            result = new StringType(entry.getTitle());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CURRENT_EVENT_SUBTITLE:
+                            entry = con.getEpgEvent(SVDRPEpgEvent.TYPE.NOW);
+                            result = new StringType(entry.getSubtitle());
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CURRENT_EVENT_DURATION:
+                            entry = con.getEpgEvent(SVDRPEpgEvent.TYPE.NOW);
+                            result = new QuantityType<>(entry.getDuration(), Units.MINUTE);
+                            break;
+                        case VDRBindingConstants.CHANNEL_UID_CURRENT_EVENT_BEGIN:
+                            entry = con.getEpgEvent(SVDRPEpgEvent.TYPE.NOW);
+                            result = new DateTimeType(LocalDateTime
+                                    .ofInstant(entry.getBegin().toInstant(), timeZoneProvider.getTimeZone())
+                                    .atZone(TimeZone.getDefault().toZoneId()));
```

Comment:
The semantic of your code is "take the VDR time and treat it as it was the timezone configured within OH and convert it to the system default time". But what you want is to convert it to the timezone configured within OH.

This suggestion assumes that the time, VDR reports, has the same timezone as the operating system OH runs on.
```suggestion
                            result = new DateTimeType(LocalDateTime
                                    .ofInstant(entry.getBegin().toInstant(), ZoneId.systemDefault())
                                    .atZone(timeZoneProvider.getTimeZone()));
```

Reply:
Thanks, removed all old java date time api.